wagtail/admin/templates/wagtailadmin/shared/breadcrumb-next.html

```diff
@@ -0,0 +1,67 @@
+{% load i18n wagtailadmin_tags %}
+
+{% with breadcrumb_link_classes='w-flex w-justify-start w-items-center w-text-grey-600 w-no-underline w-outline-offset-inside hover:w-underline hover:w-text-primary' breadcrumb_list_classes='w-overflow-hidden w-transition w-duration-300 w-whitespace-nowrap w-flex-shrink-0' icon_classes='w-w-4 w-h-4 w-mr-3' %}
+    {# Breadcrumbs are visible on mobile by default but hidden on desktop #}
+    <button data-toggle-breadcrumbs  {# handled in page-editor.js #}
+        class="w-box-border w-ml-0 w-p-4 w-h-full w-bg-transparent w-text-grey-400 w-transition hover:w-scale-110 hover:w-text-primary w-outline-offset-inside"
+        aria-label="{% trans 'Toggle breadcrumbs' %}"
+        aria-expanded="false"
+    >
+        {% icon name="dots-horizontal" class_name="w-w-4 w-h-4" %}
+    </button>
+
+    <div class="w-relative w-h-full w-bg-grey-50 w-top-0 w-z-20 w-flex w-items-center w-flex-row w-overflow-x-auto w-transition w-duration-300">
+        <nav class="w-flex w-items-center w-flex-row w-overflow-x-auto u-scrollbar-thin"
+            aria-label="{% trans 'Breadcrumb' %}">
+            <ol class="w-flex w-flex-row w-justify-start w-items-center w-pl-0 w-gap-2 sm:w-gap-0 sm:w-space-x-2 w-mr-4">
+                {% for page in pages %}
+                    {% if page.is_root %}
+                        {# Root section is shown as a 'site' icon in place of the title #}
```

Comment:
Is it intentional we’re now displaying "Root" and not having an icon? If so we should remove this comment.

Reply:
I believe "Root" might be the default now but i'll confirm this with Ben & Phil

Reply:
Cool! Then we just need to remove this comment.